### PR TITLE
Add section on Meetings to Work Mode

### PR DESCRIPTION
**Pull request by travisleithead**
Rough-draft of a new section describing the WHATWG approach to meetings.

I took this as an action from the 2021-11-03 SG meeting.

### CODE REVIEW

File: Working Mode.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ The editor has the following responsibilities, all of which can be delegated to
    * Only add features that have sufficient implementer interest.
    * Align existing features with implementations (e.g., via removal if implementer interest waned).
 
+## Meetings
```
Comment (annevk) [inline Working Mode.md:35]: Given that we don't generally do them, perhaps we should move this down, e.g., before "Anchor permanence"?

File: Working Mode.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ The editor has the following responsibilities, all of which can be delegated to
...
+
+The WHATWG doesn't generally run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liase with other groups.
+
+Requests for meetings are to be filed as issues against the relevant workstream. To allow maximum participation in a meeting, meetings should be announced early and provide an option for remote participation if possible. Transcripts, minutes, or recordings of a meeting should be made available shortly after the meeting concludes (linking from the original meeting request issue is a good way to do this).
```
Comment (annevk) [inline Working Mode.md:39]:
```suggestion
Announcements or requests for meetings are to be filed as issues against the relevant Workstream. To allow maximum participation in a meeting, meetings should be announced early and provide an option for remote participation if possible. Transcripts, minutes, or recordings of a meeting should be made available shortly after the meeting concludes (linking from the original meeting request issue is a good way to do this).
```

**Review by annevk — COMMENTED** [review]
Thanks! This generally looks good to me.

@domenic @past thoughts?

File: Working Mode.md (hunk 2 above)
Comment (domenic) [reply Working Mode.md:39]: relevant workstream -> relevant Living Standard repository?

**Review by domenic — APPROVED** [review]
LGTM with nit

File: Working Mode.md (hunk 2 above)
Comment (travisleithead) [reply Working Mode.md]: Added "Announcements or..."

For "workstream", since "Living Standard repository" is more specific, I took that suggestion.

**Review by othermaciej — APPROVED** [review]

File: Working Mode.md [hunk 3]
```diff
@@ -143,6 +143,14 @@ NOTE: A standard is a tool towards convergence, and changing a standard can ofte
 
 Implementation disagreement should not result in implementation-defined behavior or optional features.
 
+## Meetings
+
+The WHATWG doesn't generally run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liase with other groups.
```
Comment (TimothyGu) [inline Working Mode.md:148]: (typo)

```suggestion
The WHATWG doesn't generally run any regular meetings. However, as circumstances arise, meetings can be helpful to resolve difficult issues, or to liaise with other groups.
```

Comment (foolip) [reply Working Mode.md:148]: Since this PR is already merged I've sent https://github.com/whatwg/sg/pull/178. Thanks @TimothyGu!

**Pull request merged**